polyfactory/constants.py

```diff
@@ -39,3 +40,5 @@
 }
 
 IGNORED_TYPE_ARGS: Set = {Ellipsis}
+
+DEFAULT_RANDOM = Random()
```

Comment:
We could also do a hacky thing and use the `random._inst` instance, but I think this is better